src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -77,5 +77,19 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// Gets or sets the lease options for this particular DocumentDB Trigger. Overrides any value defined in <see cref="DocumentDBConfiguration" /> 
         /// </summary>
         public ChangeFeedHostOptions LeaseOptions { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Only applies to lease collection.
+        /// If true, the database and collection for leases will be automatically created if it does not exist.
+        /// </summary>
+        public bool CreateIfNotExists { get; set; } = false;
```

Comment:
Since this only applies to leases, I'd rather be verbose in the naming here -- `CreateLeaseCollectionIfNotExists`.